doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -296,53 +368,74 @@ Masking
 =======
 Documentation of _mask.py (part 1)
```

Comment:
Can probable delete this line?

Reply:
@mattiarighi do you still need these placeholders for paper stuffs?